scout/server/blueprints/variants/controllers.py

```diff
@@ -203,15 +203,16 @@ def get_manual_assessments(variant_obj):
     return assessments
 
 
-def str_variants(store, institute_obj, case_obj, variants_query, page=1, per_page=50):
+def str_variants(
+    store, institute_obj, case_obj, variants_query, variant_count, page=1, per_page=50
```

Comment:
:+1: Yep, exactly the same as #2166.

Reply:
I see. The "pagination" in the title was a bit of an understatement 😆